MvvmCross-Forms/MvvmCross.Forms.Uwp/Views/MvxFormsUwpViewPresenter.cs

```diff
@@ -129,5 +130,10 @@ private bool TryShowPage(MvxViewModelRequest request)
         public override void Close(IMvxViewModel toClose)
         {
         }
+
+        public virtual bool ClosePlatformViews()
+        {
+            return false;
```

Comment:
If the feature isn't implemented here can we leave a trace to make it easier to discover?

Reply:
Sure